### PR TITLE
chore: migrate to `@nuxt/module-builder`

### PR DESCRIPTION
**Pull request by benjamincanac**
## Types of changes

- [ ] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes (if not applicable, please state why)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,17 +29,17 @@
...
-    "standard-version": "^9.3.2",
-    "typescript": "^4.5.4",
-    "unbuild": "^0.6.7"
+    "standard-version": "^9.3.2"
   },
   "publishConfig": {
     "access": "public"
+  },
+  "build": {
+    "externals": ["vue", "nuxt3"]
```
Comment (pi0) [inline package.json:43]: This has to be added to module-builder I guess (for future reference: I would gladly appreciate if you ping or create a small issue when using workarounds for build config)

Comment (pi0) [reply package.json:43]: Fixed in @nuxt/module-builder@0.1.7

Comment (benjamincanac) [reply package.json:43]: Sure, I will create an issue with all the errors I encountered without this.

Comment (benjamincanac) [reply package.json:43]: https://github.com/nuxt/module-builder/issues/19

File: src/module.ts [hunk 2]
```diff
@@ -78,17 +77,3 @@ export default defineNuxtModule<ModuleOptions>({
     })
   }
 })
-
-declare module '@nuxt/schema' {
```
Comment (benjamincanac) [inline src/module.ts:82]: @pi0 Is `@nuxt/module-builder` taking care of `publicRuntimeConfig`, is it safe to remove?

Comment (pi0) [reply src/module.ts:82]: Not at the moment but totally open to introducing shortcut support like for type exports.

**Pull request merged**